modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/components/Comments.es.js

```diff
@@ -36,7 +35,10 @@ export default withRouter(
 		showNewComment,
 		showNewCommentChange,
 	}) => {
-		const [comment, setComment] = useState('');
+		const commentRef = useRef('');
+
+		const [cleanComment, changeToCleanComment] = useState(false);
```

Comment:
If we use the methodology I commented below, we wouldn't need this state anymore